src/components/auth.tsx

```diff
@@ -50,6 +55,19 @@ export default function Auth() {
     }
   }, [user, router])
 
+  useEffect(() => {
+    const qsError = searchParams.get("error")
+    if (qsError) {
+      setError(qsError)
+    }
+  }, [searchParams])
+
+  useEffect(() => {
+    if (error) {
+      toast.error(error)
+    }
+  }, [error])
+
```

Comment:
@robin-turnkey since we have the error stored in the searchParams do we need an extra state variable for the error or could we simplify it by doing this
```suggestion
useEffect(() => {
  const qsError = searchParams.get("error")
  if (qsError) {
    toast.error(qsError)
  }
}, [searchParams])
```